docs/pages/dsHDMIIn/ds-Hdmi-In_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,216 @@
+|3|Verify the HdmiIn Status change with callback|Select the HdmiInput port and check the callback is triggered when the status change occurs(i.e like isPresented, activeport) and  check the callbacks is triggered|`dsHdmiInRegisterStatusChangeCB()`|`NA`|`Y`|
+|4|Verify the HdmiIn Video Mode update  with callback|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,`DolbyVision`,`SDR`,...) and check the callbacks is triggered when we change the resolution|`dsHdmiInRegisterVideoModeUpdateCB()`|`NA`|`Y`|
+|5|Verify the HdmiIn Allm change  with callback|Change the allm option to TRUE/FALSE after connecting game controller on 4k supported panel and check the callbacks is triggered when the allm status change occurs|`dsHdmiInRegisterAllmChangeCB()`|`NA`|`Y`|
+|6|Verify the  HdmiIn Audio Video lateny with callback|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,`DolbyVision`,`SDR`,...) and check the callbacks is triggered when there is latency change|`dsHdmiInRegisterAVLatencyChangeCB()`|`NA`|`Y`|
+|7|Verify the HdmiIn AVI content change with callbacks|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,`DolbyVision`,`SDR`,...) and check the callbacks is triggered there is AVI content change|dsHdmiInRegisterAviContentTypeChangeCB()|`NA`|`Y`|
+|8|Get HdmiIn status of the selected port|Select the Hdmi Input and verify the status whether selected port active or not|`dsHdmiInSelectPort()`,`dsHdmiInGetStatus()`|`Y`|`Y`|
+|9|Scale HdmiIn video of the selected port|Select the Hdmi Input and scale the video on the selcted port verify video scaled or not|`dsHdmiInScaleVideo()`|`Y`|`Y`|
+|10|Check and verify Zoom mode selected|Select the Zoom mode from the available inputs and verify its set or not|`dsHdmiInSelectZoomMode()`|`Y`|`NA`|
+|11|Get Current video mode of slected input port|Play the video from the selected Hdmi Input port and get the current video mode|`dsHdmiInGetCurrentVideoMode()`|`Y`|`Y`|
+|12|Check and verify the slected port is ARC|Select the HdmiInput port and check whether the selected port ARC or not|`dsIsHdmiARCPort()`|`NA`|`Y`|
+|13|Get and verify the edid of selected port|Select the HdmiInput and get the edid for that particular Input port|`dsGetEDIDBytesInfo()`|`NA`|`Y`|
+|14|Get and verify the spdinfo of selected port|Select the HdmiInput and get the spdinfo for that particular Input port|`dsGetHDMISPDInfo()`|`NA`|`Y`|
+|15|Set and verify the edid version on selected port|Set the edid version and verify by retrieving the edid version|`dsSetEdidVersion()`,`dsGetEdidVersion()`|`NA`|`Y`|
+|16|Get the allm status of selected port|Select the HdmiInput and get the allm status for that particular Input port|`dsGetAllmStatus()`|`NA`|`Y`|
+|17|Get the SupportedGame features list |Lists all the supported game features list on sink device|`dsGetSupportedGameFeaturesList()`|`NA`|`Y`|
```

Comment:
Does the game features list involve stream playback, or does it involve reading the information directly? I don't think it is required for L3.

Reply:
actually it list the supported features thats it , if required then we will keep other wise will remove that.

Reply:
done